src/modules/MouseWithoutBorders/App/Class/Common.Helper.cs

```diff
@@ -302,6 +302,8 @@ internal static void RunDDHelper(bool cleanUp = false)
 
             IntPtr h = (IntPtr)NativeMethods.FindWindow(null, Common.HELPER_FORM_TEXT);
 
+            var processes = Process.GetProcessesByName(HelperProcessName);
```

Comment:
Why are we checking for the list of processes before creating the process? Let's have this call closer to where it was originally.